symja_android_library/matheclipse-external/src/main/java/tech/tablesaw/plotly/components/Figure.java

```diff
@@ -122,7 +122,7 @@ public String asJavascript(String divName) {
     } catch (IOException e) {
       throw new UncheckedIOException(e);
     }
-    return writer.toString().replaceAll("\\r\\n", "\n");
```

Comment:
I changed this to the state before commit 73068a6a4a08f94469abe3ebfa7eca5a5cf3b18d because it can be handled in the test cases, so at least for the tests it not necessary to change it.

Or did you change it intentionally to because it is generally required to replace `"\r\n"` by `"\n"`